### DISCUSSION TITLE
［建议］希望能支持社交账号登录

### DISCUSSION THREAD

**Discussion by AkarinLiu** [walinejs/waline#908] [Ideas]
能支持的社交账号只有github，希望能支持其他社交账号，比如微信，微博，QQ等社交账号

**Comment by CCKNBC** [comment]
@LiuShaoYeCC

**Comment by eallion** [comment]
没做过账号系统，有没有交叉关联泄露隐私的风险？

**Comment by lizheming** [comment]
> 没做过账号系统，有没有交叉关联泄露隐私的风险？

这些网站的 OAuth 登录都做的很成熟了，通过 OAuth 登录拿到对应网站的个人信息都是我们需要的基础信息，而且流程很规范。

**Comment by AkarinLiu** [comment]
不希望只局限于Github账号，微信QQ等也要支持登录啊！

**Comment by lizheming** [comment]
该 issue 已标记  enhancement 可以等后面实现了再关闭该 issue @LiuShaoYeCC 😃

**Comment by gtn1024** [comment]
期待第三方（QQ 微信等）社交平台登录🙉

**Comment by Lfhsheng** [comment]
哔哩哔哩登录期待一下😏

**Comment by lizheming** [comment]
@Lfhsheng 前几天正好想到了这个，不过 B 站开发者注册似乎比较麻烦，只能试试了

**Comment by Lfhsheng** [comment]
@lizheming  [哔哩哔哩开放平台](https://openhome.bilibili.com/doc/4/eaf0e2b5-bde9-b9a0-9be1-019bb455701c)
加油
